src/drawer.canvasentry.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import style from './util/style';
-import getId from './util/get-id';
+import getId from './util/get-id.ts';
```

Comment:
Need help to fix this. you should be able to import typescript files without the extension